assets/sass/_typography.scss

```diff
@@ -122,7 +122,9 @@ $bold-font-weight: 600;
 $heading-font-weight: 700;
 
 // Font sizes
+$smaller-font-size: 65%;
```

Comment:
Might want to go to 62.5%, since our base font size is effectively 16px atm — this sets $smaller-font-size then to 10px.

I have some concerns about adding such a small size generally, but yea, let’s revisit in a typography spike sometime.